docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,36 +1,28 @@
-= Getting Started
+= Contracts
 
-*OpenZeppelin is a library for secure smart contract development.* It provides implementations of standards like ERC20 and ERC721 which you can deploy as-is or extend to suit your needs, as well as Solidity components to build custom contracts and more complex decentralized systems.
+*A library for secure smart contract development.* Build on the shoulders of giants.
```

Comment:
Let's tone this down a bit. 😬 
```suggestion
*A library for secure smart contract development.*
```

Reply:
While this was a bit of a joke, I did like the tone and idea behind the message. Can you think of an alternative that you feel comfortable with?

Reply:
Something along the lines of "Build on a solid foundation of..."

Reply:
Build on a solid foundation of community-vetted code?